wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1675,13 +1668,14 @@ class OuterScopeShadowingViolation(ASTViolation):
     """
     Forbids to shadow variables from outer scopes.
 
-    We check function, method, and module scopes.
-    While we do not check class scope. Because class level constants
+    We check the function, method, and module scopes.
```

Comment:
I'd say this makes slightly more sense without "the". However, both are perfectly comfortable to read.